src/components/EditOrganization/index.tsx

```diff
@@ -113,18 +112,10 @@ export default function OrganizationView({ organization, reloadOrganizationData
     } else {
       const response = await updateOrganization(organizationRecord);
       if (response.requestSucceeded) {
-        setSnackbar({
-          type: 'toast',
-          priority: 'success',
-          msg: 'Changes saved',
-        });
+        snackbar.toastSuccess('Changes saved');
```

Comment:
This should have use string files, but I know it was like that before haha. So no need to change it now

Reply:
yeah, i saw that. i will move it to strings.